src/main/plugins/org.dita.html5/xsl/topic.xsl

```diff
@@ -1990,8 +1990,12 @@ See the accompanying LICENSE file for applicable license.
   <xsl:template name="spec-title-nospace">
    <xsl:if test="@spectitle"><div style="margin-bottom: 0;"><strong><xsl:value-of select="@spectitle"/></strong></div></xsl:if>
   </xsl:template>
-    
+
+  <!-- Deprecated since 3.7, use "copyright" mode instead -->
```

Comment:
However, we cannot rewrite current uses of

```xml
<xsl:call-template name="copyright"/>
```

into

```xml
<xsl:apply-templates select="." mode="copyright"/>
```

Because that would not longer work with current plugins that have overridden `copyright` named template.